README.md

```diff
@@ -1,4 +1,6 @@
-[![Build Status](https://travis-ci.org/catalyst/moodle-tool_crawler.svg?branch=master)](https://travis-ci.org/catalyst/moodle-tool_crawler)
+# moodle-tool_crawler
+
+![GitHub Workflow Status (branch)](https://img.shields.io/github/workflow/status/catalyst/moodle-tool_crawler/ci/MOODLE_310_STABLE)
```

Comment:
Also it depends if you want it to display the default `build: status` or `ci: status` (using a label=ci request paramater as per the v2 docs)